docs/tutorial.rst

```diff
@@ -55,7 +54,7 @@ Each statistical test consist of one kpi, treatment and control variant names, a
 
     kpi = KPI('normal_same')
     variants = Variants(variant_column_name='variant', control_name='B', treatment_name='A')
-    test = StatisticalTest(kpi=kpi, features=[], variants=variants)
+    test = StatisticalTest(data=data, kpi=kpi, features=[], variants=variants)
```

Comment:
ahaha you made it the first para in the end!

Reply:
@shansfolder yes, I decided to keep the priority of the parameters: first and most important is data, then kpis, then features (we can do testing without them).